recipes/cymysql/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cymysql" %}
+{% set version = "0.9.14" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b2a20c6d5373a16590b75f149a7cb2215a6d3bb2fc856b1b75f437ccf1e716d4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.3
+  run:
+    - python >=3.3
```

Comment:
Don't you need ``cython``?

As the summary says "Python MySQL Driver using **Cython**"

Reply:
Actually, I read the ``setup.py`` and it seems you need ``cython`` on the ``host`` section to build some modules.

Reply:
@marcelotrevisani thanks, cython has been added to host.